lib/cisco_node_utils/vxlan_global.rb

```diff
@@ -29,17 +30,19 @@ def initialize(instantiate=true)
     end
 
     def enable
-      config_set('vxlan_global', 'feature', state: '')
+      Feature.fabric_enable unless Feature.fabric_enabled?
+      config_set('feature', 'fabric_forwarding', state: '')
     end
 
     def disable
-      config_set('vxlan_global', 'feature', state: 'no')
+      config_set('feature', 'fabric_forwarding', state: 'no') if
+        Feature.fabric_enabled?
```

Comment:
We've elected to never turn features off anymore. So just remove this 'no' call altogether.

The dup_host_mac_detection_default can stay.